src/index.js

```diff
@@ -1 +1,32 @@
 import './style.scss';
+import newScore from './modules/newScore.js';
+import generateHtml from './modules/generateHtml.js';
+import getScores from './modules/getScores.js';
+
+const form = document.getElementById('newScore');
+const scoreContainer = document.getElementById('scores__container');
+const refreshButton = document.getElementById('refresh-button');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const name = form.elements.name.value;
+  const score = parseInt(form.elements.score.value, 10);
+  newScore(name, score);
+});
```

Comment:
- [OPTIONAL] After submitting a score, it would be good for user interaction if you cleared the form inputs for both the name and score. You could also add a message that would info the user if their scores was successfully sent to the API endpoint. 👍 😎